### ISSUE TITLE
paginator 

### ISSUE THREAD

**Issue by burhanaksendir** [CalsRanna/book_reader#2]
Hello,
I tried to try your pagination method, but I couldn't understand the value of 20 in this section:

https://github.com/CalsRanna/book_reader/blob/677fb8ed017e5377a24b62e414fffa41a4890f77/lib/src/tool/paginator.dart#L16

**Comment by CalsRanna** [comment]
Basically, it will get the small enough words after 10 times loop even your raw data counts 20000 words. So max 20 will be some proper limitations reduce calculation costs.
But if you have more words, rethink about it to increase it or change another way to paginate words.

**Comment by burhanaksendir** [comment]
@CalsRanna  Thank you for the explanation.  I tried it with a text that is RTL enabled. However, I am not getting the correct result. Some words are incorrectly split and go to the next page. Here is my code and screenshot. May I ask for your help with this issue?
 
https://gist.github.com/burhanaksendir/c7e279664b0ff20f995b247848f14a1f


At the end of the page, the word I underlined in red is incorrectly split. half of the word: 
![01](https://user-images.githubusercontent.com/8548639/230342243-8cc05983-4226-41af-ba8f-adafff944d6b.png)

![02](https://user-images.githubusercontent.com/8548639/230342257-2f83d2b5-8d93-48b7-9ef5-84cffbe6bcab.png)

Expected correct result for incorrectly split word:
![03](https://user-images.githubusercontent.com/8548639/230342002-8cdc14cd-5167-4f13-9ef1-75e6c8fecab1.png)
